### ISSUE TITLE
suite control (mutations)

### ISSUE THREAD

**Issue by oliver-sanders** [cylc/cylc-ui#135]
This issue covers the GUI side of https://github.com/cylc/cylc-flow/issues/2123.

* Auto-generate forms for suite actions (e.g. poll, insert, reload, kill, ...)
* Present information about running commands (blocked by GraphQL subscriptions and https://github.com/cylc/cylc-flow/issues/2978)

Basic design outline (take with a pinch of salt):

![gui-forms](https://user-images.githubusercontent.com/16705946/61459285-4fabe580-a964-11e9-8735-f03e645c7af9.png)

**Comment by kinow** [comment]
Brilliant! Now I know what will go under the Control option. And really helpful to have this sort of workflow diagram. Thanks!

**Comment by kinow** [comment]
Related to #2123 I think

**Comment by oliver-sanders** [comment]
Bumping to 2.0 as the blocking Cylc Flow issue (https://github.com/cylc/cylc-flow/issues/3329) is unlikely to be completed by 8.0.0 as it is more of a nice-to-have than an essential feature.
